packages/omg-js-childchain/src/rpc/rpcApi.js

```diff
@@ -46,14 +46,15 @@ async function parseResponse (resp) {
     // Need to use a JSON parser capable of handling uint256
     json = JSONBigNumber.parse(body)
   } catch (err) {
+    console.warn(err)
     throw new Error('Unknown server error')
   }
   debug(`rpc response is ${JSON.stringify(json)}`)
 
   if (json.success) {
     return json.data
   }
-  throw new RpcError(json.data)
+  throw new Error(JSON.stringify(json.data))
```

Comment:
Can we either go back to using the RpcError class, or delete it.
I'm fine with either way.

Reply:
oh forget to move back to RpcError, it was changed beacuse RpcError actually hide the error from childchain in `messages` but I saw you fixed that 👌